discord/ui/view.py

```diff
@@ -256,6 +256,7 @@ def add_item(self, item: Item[Any]) -> Self:
         """Adds an item to the view.
```

Comment:
A whitespace under this line should also be added to stay consistent with rest of the docstring's.

Reply:
Seems to be the case with `View.remove_item` and `View.clear_items` too. I'll add one to those as well.